JavaAssignments/src/saket/swapwiththirdvaiable.java

```diff
@@ -0,0 +1,23 @@
+package saket;
+
+public class swapwiththirdvaiable {
+
+	int swap(int a, int b) {
+		int c;
+		c = a;
+		a = b;
+		b = c;
+		System.out.println("a=" + a);
+		System.out.println("b=" + b);
+
+		return 0;
```

Comment:
if you are always returning 0, it doesn't make any sense.
Here you dont need to return anything. you should have return type as void.